open_humans/settings.py

```diff
@@ -450,6 +422,7 @@ def __str__(self):
 AUTHENTICATION_BACKENDS = (
     'oauth2_provider.backends.OAuth2Backend',
     'django.contrib.auth.backends.ModelBackend',
```

Comment:
maybe add above this:
`    # ModelBackend before allauth + our User -> iexact email/username login`